src/git-mirror/git-mirror-task.spec.ts

```diff
@@ -285,20 +285,29 @@ describe("GitMirrorTask", () => {
             const pullRef2 = "9eb061c8e73b4c0a08613710daa3f5a93d47061d refs/pull/13/head";
             const mergeRequest1 = "1b60f335a6f3a4bbf7d56a8ae3106f6b3ea77891 refs/merge-requests/10/head";
 
-            const updatedPackedRefsFileContents = "data: # pack-refs with: peeled fully-peeled sorted \\n" +
-            `${branchRef1}\\n${branchRef2}\\n${tagRef1}\\n${tagRef2}\\n`;
+            const updatedPackedRefsFileContents = "data: # pack-refs with: peeled fully-peeled sorted \n" +
+            `${branchRef1}\n${branchRef2}\n${tagRef1}\n${tagRef2}\n`;
 
             const originalPackedRefsFileContents = updatedPackedRefsFileContents +
-                `${pullRef1}\\n${pullRef2}\\n${mergeRequest1}\\n`;
+                `${pullRef1}\n${pullRef2}\n${mergeRequest1}\n`;
+
+            const getDefaultGitCloneDirectory = GitMirrorTask.prototype.getDefaultGitCloneDirectory;
```

Comment:
The underlying private field utilized when the ref review is invoked is set by the constructor, so to control the default value (without incurring a broader refactor of the app code) we have to be able to control what this function call returns on the prototype itself.

However, we also need to be able to "unset" that stub value so that other tests can have access to the original function, so we grab that here and then use it in the test suite exit point to "restore"